pom.xml

```diff
@@ -241,7 +241,7 @@
         <maven.plugin.jacoco.version>0.8.11</maven.plugin.jacoco.version>
         <maven.plugin.scalastyle.version>1.0.0</maven.plugin.scalastyle.version>
         <maven.plugin.shade.version>3.5.2</maven.plugin.shade.version>
-        <maven.plugin.silencer.version>1.7.13</maven.plugin.silencer.version>
+        <maven.plugin.silencer.version>1.7.17</maven.plugin.silencer.version>
```

Comment:
Why we have to update the silencer version?

Reply:
each version of silencer has a strictly supported Scala version list, it's common to update with Scala version together
https://mvnrepository.com/artifact/com.github.ghik/silencer-plugin

Reply:
Cool. SGTM. Thanks for the explanation.